Sources/Datadog/Core/DatadogCoreProtocol.swift

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
How about putting these classes in separate `Datadog/DatadogCore` (or whatever) folder? Even though this won't yet be a standalone module, it would establish a conventional module. Just like `Datadog/Core` is a conventional core today, with no dependencies to other folders (other conventional modules).